docs/concepts/crds/function/index.md

```diff
@@ -5,23 +5,23 @@ menu:
     identifier: function-overview
     name: Function
     parent: crds
-    weight: 30
+    weight: 35
 product_name: kubestash
 menu_name: docs_{{ .version }}
 section_menu_id: concepts
 ---
 
-> New to Stash? Please start [here](/docs/concepts/README.md).
+> New to KubeStash? Please start [here](/docs/concepts/README.md).
 
 # Function
 
 ## What is Function
 
-A complete backup or restore process may consist of several steps. For example, in order to backup a PostgreSQL database we first need to dump the database and upload the dumped file to a backend. Then we need to update the respective`Repository` and `BackupSession` status and send Prometheus metrics. In Stash, we call such individual steps a `Function`.
+In KubeStash, a complete task like backup or restore is referred to as a `Function`.
 
-A `Function` is a Kubernetes `CustomResourceDefinition`(CRD) which basically specifies a template for a container that performs only a specific action. For example, `postgres-backup-*` function only dumps and uploads the dumped file into the backend where `update-status` function updates the status of respective `BackupSession` and `Repository` and sends Prometheus metrics to pushgateway based on the output of `postgres-backup-*` function.
+A `Function` is a Kubernetes `CustomResourceDefinition`(CRD) which basically specifies a template for a container that performs only a specific action or task. For example, `postgres-backup` function only take backup of PostgreSQL Database.
```

Comment:
When backing up or restoring an application, there can be multiple tasks involved. For example, with a PostgreSQL database, you might need to do a manifest backup and a logical backup.In KubeStash, each of these tasks is performed using a dedicated `Function`.